src/firebase.py

```diff
@@ -53,6 +54,42 @@
 _firebase_app: Optional[App] = None
 _firebase_app_lock = threading.Lock()
 
+firebase_cred = credentials.Certificate(os.path.abspath(os.path.join("resources", "explo-dev-firebase-adminsdk-nlwq7-706a6b3fdd.json")))
```

Comment:
I would prefer to have the path relative to the location of the source file, i.e. relative to `__file__`,
cf.
```
FIREBASE_CRED_PATH = os.path.join(os.path.dirname(__file__), "..", "resources", "explo-dev-firebase.json")
```